### PR TITLE
🎓:bomb: Move Logic Content

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
1. Move the topic more on logic before conditionals
2. Remove unnecessary sections (passing functions as parameters, building more complex programs (this is covered in the subsequent topic)).  

### Why
I think things will flow better this way. 

### Additional Notes
There are no real content changes yet and the sections were moved with just copy/paste. The actual massaging and cleaning up will happen in another PR.

### CODE REVIEW

File: site/topics/booleans/booleans.rst [hunk 1]
```diff
@@ -1,6 +1,59 @@
...
+.. admonition:: Activity
+
+    Write out the truth tables for the logical operations **AND**, **OR** and **NOT**.
+
+    Don't know what a 'logical operator' or 'truth table' is? No problem, Ask:
+
+    * Wikipedia
+    * Google
+    * Your neighbour
+    * Just don't ask me!
```
Comment (twentylemon) [inline site/topics/booleans/booleans.rst:23]: Could maybe use some toning down.

File: site/topics/booleans/booleans.rst [hunk 2]
```diff
@@ -1,6 +1,59 @@
...
+* Let's try some comparison:
+    >>> 19 == 87
+    False
+
+    >>> 5 == 5
+    True
+
+* Note that ``==`` is *comparison* while ``=`` is *assignment*. They are not the same! Python will punish you if you forget this!
+    * You'll all mess this up eventually
+    * Just be thankful this isn't C/C++
```
Comment (twentylemon) [inline site/topics/booleans/booleans.rst:37]: Could frame this more positively. Mistakes need to be made in order to learn from them. Mentioning c/cpp is unnecessary.

**Review by twentylemon — CHANGES_REQUESTED** [review]

File: site/topics/booleans/booleans.rst (hunk 1 above)
Comment (jameshughes89) [reply site/topics/booleans/booleans.rst:23]: I'll address these when I go for the actual rework of the content. This PR is just moving things around.

Comment (twentylemon) [reply site/topics/booleans/booleans.rst:23]: Ack.

**Review by twentylemon — APPROVED** [review]

**Pull request merged**